### PR TITLE
Don't skip windows, add toolchain to build reqs

### PR DESCRIPTION
**Pull request by pmlandwehr**
To get the [feedstock for `paramiko`](https://github.com/conda-forge/paramiko-feedstock/pull/10) up to the latest version, I need a windows version of `pynacl`. This is my naive attempt at getting a windows build by adding `toolchain` to the run reqs.

### CODE REVIEW

**Comment by pmlandwehr** [comment]
(I'll add here that because of issues with`conda-smithy` right now I *can't regenerate* the feedstock. and regeneration is going to be necessary to get the windows build running. So this is perhaps all futile.

**Comment by pmlandwehr** [comment]
Hah! Fixed my version of `conda-smithy`, so rerender completed.

**Comment by CJ-Wright** [comment]
I put myself on as a reviewer so I don't loose track of it.

**Comment by CJ-Wright** [comment]
@conda-forge-admin please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,30 +1,31 @@
...
   script: python setup.py install --single-version-externally-managed --record record.txt
 
 requirements:
   build:
     - python
     - setuptools
-    - cffi
+    - toolchain
+    - cffi >=1.4.1
+
```
Comment (CJ-Wright) [inline recipe/meta.yaml:24]: I think we most likely need to also use the libsodium that is in conda-forge if possible

**Pull request closed without merging**